client/src/components/RoadmapItem/index.tsx

```diff
@@ -5,22 +5,39 @@ interface RoadmapItemProps {
   columns: number;
   index: number;
   length: number;
+  exceedsLeft: boolean;
+  exceedsRight: boolean;
+  handleDragStart: () => void;
 }
 
-const RoadmapItem = ({ columns, index, length }: RoadmapItemProps) => {
+const RoadmapItem = ({
+  columns,
+  index,
+  length,
+  exceedsLeft,
+  exceedsRight,
+  handleDragStart,
+}: RoadmapItemProps) => {
   return (
     <S.Container columns={columns}>
       {[...Array(columns)].map((_, i) => {
         if (i === index)
           return (
             <S.Bar key={i}>
-              <S.FrontHandle />
+              {exceedsLeft ? undefined : (
+                <S.FrontHandle draggable="true" onDragStart={handleDragStart} />
+              )}
+              {length === 0 && !exceedsRight ? (
+                <S.RearHandle draggable="true" onDragStart={handleDragStart} />
+              ) : undefined}
```

Comment:
```suggestion
              {length === 0 && !exceedsRight && (
                <S.RearHandle draggable="true" onDragStart={handleDragStart} />
              )}
```
이 부분도 위와 같은 방식으로 변경할 수 있을 것 같습니다. `length === 0` 을 체크하시는 이유가 뭔지 궁급합니다.